pootle/apps/pootle_store/util.py

```diff
@@ -118,3 +118,11 @@ def get_change_str(changes):
         return ", ".join(res)
 
     return "no changed"
+
+
+def parse_pootle_revision(store):
+    if hasattr(store, "parseheader"):
+        pootle_revision = store.parseheader().get("X-Pootle-Revision",
+                                                  None)
+        if pootle_revision is not None:
+            return int(pootle_revision)
```

Comment:
however i dont think we want to replace the not isinstance(store_revision, int) as the rest of the condition relies on the first value being an int